runtimes/runtimes/auth/auth.ts

```
@@ -137,6 +143,18 @@ export class Auth {
         })
     }
 
+    private async handleBearerCredentialsMetadata(metadata?: ConnectionMetadata) {
+        if (metadata) {
+            this.connectionMetadata = metadata
```

Guess would be good to do more validation of incoming `metadata` object, since client could send whatever. Also not sure if we should return an error to client here, but at least logging something would be good.

---

All the fields in `metadata` are optional. Given that I'm not sure I should validate any other fields. If client send the metadata, he defined it already, even if it's empty.